Lib/test/test_tools/test_i18n/__init__.py

```diff
@@ -0,0 +1,6 @@
+import os.path
+from test import support
+
+
+def load_tests(*args):
+    return support.load_package_tests(os.path.dirname(__file__), *args)
```

Comment:
Is this necessary? Maybe use a subdirectory for data files only?

Reply:
It's not, I'll remove it :)